### ISSUE TITLE
'mimosa build --errorout' exits with status code 0 (success)

### ISSUE THREAD

**Issue by chrylis** [dbashford/mimosa#422]
If Mimosa is invoked with the `--errorout` option to abort processing when an error is detected, it shouldn't return 0 to the shell.

**Comment by dbashford** [comment]
This usually happens because some sort of error isn't reporting itself as one that should abort processing.  Which error are you encountering that is causing this issue?

**Comment by chrylis** [comment]
No, I'm getting a correct error message printed to the console, but then the shell still thinks the build succeeded. Append random nonsense to the config file and try. You'll get an immediate error and abort, but Mimosa will return success to the shell.

**Comment by dbashford** [comment]
My point is it doesn't always do this.  That is one case it does.  Are there others?

**Comment by chrylis** [comment]
I don't know about that. Are there duplicate code paths that `-e` switches on? I would have expected a corrupted config file to produce a non-zero exit even without `-e`.

**Comment by dbashford** [comment]
Errors need to opt in to allow that flag to error out the process.  As time has passed I have caught be occasional error that wasn't opting in when it should.

Agree that this should error out with a non 0 regardless.

**Comment by chrylis** [comment]
Got it, and I'll try to keep an eye out for any issues like that.

**Comment by dbashford** [comment]
Released v`2.3.26`, should resolve the specific issue you brought up.  Should you run across another, please do open up another issue for it.  Thanks!

```
$ mimosa watch -s
...
10:20:28 - FATAL - FATAL: Improperly formatted configuration file mimosa-config.js: SyntaxError: Unexpected identifier
$ echo $?
1
```
